packages/web-react/src/components/Field/constants.ts

```diff
@@ -0,0 +1 @@
+export const ALERT_ROLE = 'alert';
```

Comment:
**suggestion (if-minor):** IMHO, we should use a prefix like `a11y` so the constant will be more readable.
```suggestion
export const A11Y_ROLE_ALERT = 'alert';
```